haiku-apps/bescreencapture/bescreencapture-2.1.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="A screen recorder utility"
+DESCRIPTION="BeScreenCapture is a screen recorder utility for Haiku.
+It allows you to record what happens on your screen, then save it \
+to any media format supported in Haiku.
+BeScreenCapture can record either the entire screen, or just a section you \
+select."
+
+SUMMARY_inputfilter="Shortcut handler for BeScreenCapture"
+DESCRIPTION_inputfilter="Input Server Addon for BeScreenCapture. Allows the \
+user to launch BeScreenCapture and start/stop recording using a keyboard \
+combination (CTRL-COMMAND-SHIFT + R)."
+
+HOMEPAGE="https://github.com/jackburton79/bescreencapture"
+SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v2.1.tar.gz"
+CHECKSUM_SHA256="5b3ff575fcfe7519556fcd3d9487ef5491ead656f1cffd79b78dfcb3a52332f8"
+SOURCE_FILENAME="bescreencapture-2.1.tar.gz"
+LICENSE="
+	BSD (3-clause)
```

Comment:
> I updated the recipe with the requested changes.

All the changes are OK, but you forgot to fix the ordering of a few variables:
* COPYRIGHT should go right after HOMEPAGE.
* LICENSE should go right after COPYRIGHT.
* REVISION should go right after LICENSE.

There are a couple of empty lines to add or remove:
* It is recommended to **add** a blank line between SOURCE_FILENAME and ARCHITECTURES.
* It is recommended to **remove** the blank line between PROVIDES and PROVIDES_inputfilter.

All the rest is OK ;-)